cli/src/commit_templater.rs

```diff
@@ -1253,6 +1253,22 @@ fn builtin_commit_methods<'repo>() -> CommitTemplateBuildMethodFnMap<'repo, Comm
             Ok(out_property.into_dyn_wrapped())
         },
     );
+    map.insert(
+        "conflicted_files",
+        |_language, _diagnostics, _build_ctx, self_property, function| {
+            function.expect_no_arguments()?;
+            let out_property = self_property.and_then(|commit| {
+                let tree = commit.tree();
+                let entries: Vec<_> = tree
+                    .conflicts()
```

Comment:
FYI, I may make this `.conflicts()` call slower in the near future in order to address #4152. I don't think it'll be very noticeable, and of course it will only affect commits with conflicts. I'm mentioning this because I think the performance of `.conflicts()` currently doesn't matter that much because we only call it for your current conflict. As I said, this is just FYI; I'm not asking for any changes.

Reply:
Ack.